ImageProcessing/src/image/geometric/Translate.md

```diff
@@ -0,0 +1,15 @@
+# image.geometric.translate
+
+The translate operator performs a geometric transformation which maps the position of each picture element in an input image into a new position in an output image. The image element located at (x1,y1) in the original is shifted to a new position (x2,y2) in the corresponding output image by displacing it through a user-specified translation (Ox,Oy).
+
```

Comment:
From what I understand, the reason why user defined was chosen is because the algorithm in this webpage included it, (we were working towards affine transformation which is a combination of the geometric operators on that page) http://homepages.inf.ed.ac.uk/rbf/HIPR2/translte.htm 
user defined just means that the offset of where the image starts. ill upload an image later of it working.